test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -868,6 +870,34 @@ public void ValidateObjectTypesOptionswithEnum()
             }
         }
 
+        // <summary>
+        /// Verify that publish profile gets created and saved
+        /// </summary>
+        [Test]
+        public async Task ValidateSavePublishProfile()
+        {
+            DacFxService service = new DacFxService();
+            string folderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "DacFxTest");
+            string profileFilePath = Path.Combine(folderPath, string.Format("validateSavePublishProfile.publish.xml"));
+
+            var savePublishProfileParams = new SavePublishProfileParams
+            {
+                ProfilePath = profileFilePath,
+                DatabaseName = "testDb",
+                ConnectionString = "testConnString",
+                SqlCommandVariableValues = new Dictionary<string, string>()
+                    {
+                        { "testvar", "testval" }
+                    }
+            };
+
+            MockRequest<ResultStatus> requestMock = new();
+
+            await service.HandleSavePublishProfileRequest(savePublishProfileParams, requestMock.Object);
+
+          VerifyAndCleanup(profileFilePath);      // verify file gets created
```

Comment:
Can you add verification that the file contains all the settings?  As is, I think the test doesn't cover the Tools Service code that does the copying, and would pass if that was broken.

Reply:
Added additional content verification.